src/plays/devblog/Devblog.js

```diff
@@ -0,0 +1,23 @@
+import PlayHeader from 'common/playlists/PlayHeader';
+import './styles.css';
+import App from './App';
+
+// WARNING: Do not change the entry componenet name
+function Devblog(props) {
+  // Your Code Start below.
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className="play-details-body">

```

Comment:
@Coderamrin I have the same comment that given for the last play. Add a unique style and use it for all in your CSS file. Thanks!